prov/efa/src/rdm/rxr_msg.c

```diff
@@ -635,6 +645,9 @@ struct rxr_op_entry *rxr_msg_alloc_rx_entry(struct rxr_ep *ep,
 		memset(&rx_entry->desc[0], 0, sizeof(rx_entry->desc));
 
 	rx_entry->cq_entry.op_context = msg->context;
+
+	// TODO - set fi_peer_rx_entry fields required by SHM provider when allocating
```

Comment:
Yes, I plan to have it in my PR that implements get_*, we should refer to how shm uses the fi_peer_rx_entry: https://github.com/ofiwg/libfabric/blob/main/prov/shm/src/smr_progress.c#L648-L721

Reply:
My latest finding is we really do not need to set `fi_peer_rx_entry` fields in this commonly used function as owner provider (EFA) will not use them for now. I can set them in `get_msg/tag` after getting an rx entry.

Reply:
>Only SHM provider needs this info. EFA provider will just use rx_entry. So I didn't implement that now and added the TODO instead. I can add the info when implementing the SHM provider?

After thinking about this comment. I think I will hold this change in my owner_ops PR as well before integrating with shm. Let's keep the TODO here then.